tools/ld-dropout-correct/dropoutcorrect.cpp

```diff
@@ -75,54 +90,71 @@ void DropOutCorrect::run()
                         " drop-outs";
 
             // Analyse the drop out locations in the first field
-            QVector<DropOutLocation> firstFieldDropouts;
-            if (firstFieldMetadata[0].dropOuts.startx.size() > 0) firstFieldDropouts = setDropOutLocations(populateDropoutsVector(firstFieldMetadata[0], overCorrect));
+            QVector<QVector<DropOutLocation>> firstFieldDropouts;
```

Comment:
You can give QVector's constructor the size directly (as with std::vector), i.e. rather than writing

```
QVector<T> foo;
foo.resize(n);
```

just write

```
QVector<T> foo(n);
```

Reply:
Done